Patches/QuickSaveTweaks.cpp

```diff
@@ -119,7 +119,8 @@ void __stdcall print_no_quick_save_string(void)
 injector::hook_back<void(__cdecl*)(void)> ClearTextFun;
 void __cdecl ClearTextHook()
 {
-	UpdateMemoryAddress(GetClearTextPointer(), "\x00", 1);
+	if (*GetClearTextPointer() == 0x05)
```

Comment:
If `GetClearTextPointer()` returns nullptr it will crash here.  We should have some way to test the pointer before using it.

Reply:
Added a check to bypass it!

Reply:
Great.  Thanks @mercury501!